packages/utils/other-utils/tsconfig.json

```diff
@@ -5,5 +5,4 @@
     "outDir": "dist-tsc",
     "rootDir": "src"
   },
```

Comment:
```suggestion
  },
  "include": ["src"]
```

Reply:
Thanks! I didn't notice that I deleted this entry and I was wondering why my builds started filing out of a sudden